recipe-server/client/control_new/components/extensions/ExtensionForm.js

```diff
@@ -40,7 +37,7 @@ export class _ExtensionForm extends BaseAntForm {
 
   componentWillReceiveProps(newProps) {
     // Reset form if the extension changed
-    if (!isEqual(newProps.extension, this.props.extension)) {
+    if (!is(newProps.extension, this.props.extension)) {
```

Comment:
Oh cool, I had seen this in Immutable's testing but didn't realize it could be pulled into actual code. Disregard my `.equals` comment on previous review.